project.godot

```diff
@@ -811,7 +811,7 @@ search_in_file_extensions=PoolStringArray( "gd", "shader", "esc" )
 
 [editor_plugins]
 
-enabled=PoolStringArray( "res://addons/escoria-core/plugin.cfg", "res://addons/escoria-dialog-simple/plugin.cfg", "res://addons/escoria-ui-9verbs/plugin.cfg", "res://addons/escoria-wizard/plugin.cfg" )
+enabled=PoolStringArray( "res://addons/escoria-core/plugin.cfg", "res://addons/escoria-wizard/plugin.cfg" )
```

Comment:
This seems wrong, no? Shouldn't the dialog plugin and 9verbs plugin also be enabled?

Reply:
Fixed, I think